### PR TITLE
Adds API definition as an inline parameter when importing APIs in publisher REST Apis

### PR DESCRIPTION
**Pull request by Ketharan**
## Purpose
Provides a way to create/validate APIs providing the open API specification as the inline string value to the Publisher REST APIs

## Approach
Adds an inline form parameter so the user can provide the inline open API specification and create/validate APIS. 

## Automation tests
 - Integration tests
Tested the behavior in a locally build pack

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
java 1.8.0_251

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml [hunk 1]
```diff
@@ -1310,6 +1310,9 @@ paths:
                 additionalProperties:
                   type: string
                   description: Additional attributes specified as a stringified JSON with API's schema
+                inlineAPIDefinition:
+                  type: string
+                  description: Inline content of the swagger file
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml:1315]:
```suggestion
                  description: Inline content of the OpenAPI defintion 
```

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml [hunk 2]
```diff
@@ -1530,6 +1533,9 @@ paths:
                   type: string
                   description: OpenAPI definition as a file
                   format: binary
+                inlineAPIDefinition:
+                  type: string
+                  description: Inline content of the swagger file
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml:1538]:
```suggestion
                  description: Inline content of the OpenAPI definition 
```

**Comment by bhathiya** [comment]
@malinthaprasan please review and merge.

**Comment by malinthaprasan** [comment]
@Ketharan please update the PR as discussed
Sorry about the trouble caused because of the delay to getting this merged..

**Comment by Ketharan** [comment]
The conflicts are resolved. @malinthaprasan could you have a look.

**Review by malinthaprasan — APPROVED** [review]

**Pull request merged**